packages/palette/src/elements/Popover/Popover.tsx

```diff
@@ -40,6 +41,7 @@ export const Popover: React.FC<PopoverProps> = ({
   visible: _visible = false,
   children,
   popover,
+  denyFlipPosition,
```

Comment:
It would be better to name this the positive case 'flipPosition' and then set a default to be 'true'